lib/current-env.js

```diff
@@ -13,22 +10,8 @@ function cpu () {
   return process.arch
 }
 
-function libc (osName) {
-  // this is to make it faster on non linux machines
-  if (osName !== 'linux') {
-    return undefined
-  }
-  let family
-  const originalExclude = process.report.excludeNetwork
-  process.report.excludeNetwork = true
-  const report = process.report.getReport()
-  process.report.excludeNetwork = originalExclude
-  if (report.header?.glibcVersionRuntime) {
-    family = 'glibc'
-  } else if (Array.isArray(report.sharedObjects) && report.sharedObjects.some(isMusl)) {
-    family = 'musl'
-  }
-  return family
+function libc () {
+  return familySync() || undefined
```

Comment:
Could you cache locally the result of familySync?

As it can still be a not so fast call (sync file read operation or blocking get report) and it's done for each dependency in the tree, so it should be cached because it will never change